### PR TITLE
Meta station atmos opening up

### PR DESCRIPTION
**Pull request by MrJWhit**
## About The Pull Request

Reformats metastation atmos to a more open design (notably all pumps are past the glass wall area).
Adds layer mannifolds to the output of each gas tank.
Adds a second atmos hardsuit.
Reduces the air/o2/n2/n2o tanks in the holding area to 2 each, rather than 3 each.
Adds a neat holding area that you can go into the main atmos area, or the atmos control room.

Also fixes two minor piping issues on metastation, since I was checking for broken pipes.

## Why It's Good For The Game

I honestly really enjoy the reformatting of boxstation, and as per a 11:5 vote on the discord. I'm going to try and keep a uniqueness to each of the maps, but also adjust it so that there's more space to enjoy things.

## Changelog
:cl:
tweak: Opened up atmos a /lot/
balance: Added a second atmos hardsuit and reduced pregenerated tanks in atmos control room on metastation.
/:cl:

### CODE REVIEW

**Comment by MrJWhit** [comment]
Why is travis upset
I didn't adjust any code, why is it upset.

File: _maps/map_files/MetaStation/MetaStation.dmm [hunk 1]
```diff
@@ -54005,64 +53669,81 @@
...
 /turf/open/floor/plasteel/dark,
 /area/engine/atmos)
 "ccO" = (
-/obj/machinery/atmospherics/pipe/simple/dark/visible{
-	dir = 9
+/obj/machinery/atmospherics/pipe/simple/green/visible{
+	dir = 4
+	},
+/obj/machinery/atmospherics/components/binary/pump{
+	name = "Mix to Fuel Pipe""
```
Comment (Trilbyspaceclone) [inline _maps/map_files/MetaStation/MetaStation.dmm:53693]:
```suggestion
	name = "Mix to Fuel Pipe"
```

Comment (MrJWhit) [reply _maps/map_files/MetaStation/MetaStation.dmm:53693]: Whoops, looks like a misname, thanks

**Comment by MrJWhit** [comment]
I ask that this not be merged until this PR gets merged/discussed: https://github.com/Citadel-Station-13/Citadel-Station-13/pull/12048
I can close this if it's a worry that it will be merged without adjustments though.

**Comment by Ghommie** [comment]
From what I see on the map merge, it seems you did the same mistakes of #11947 here too.

**Comment by MrJWhit** [comment]
Those aren't mistakes, those are intentionally left open, as per /vg/'s map design.

*However*, if the playerbase doesn't like the map designs, that's totally fine, and I can fix it. So that's why I ask that it waits.

**Review by Ghommie — APPROVED** [review]
:thinking:

**Pull request merged**